Test/Core/Components/CollisionListener.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+
+namespace Duality.Tests.Components
+{
+	/// <summary>
+	/// A collision listener to expose the collision events to tests
+	/// </summary>
+	public class CollisionListener : Component, ICmpCollisionListener
```

Comment:
Rename to `CollisionEventReceiver` in order to match `InitializableEventReceiver` [here](https://github.com/AdamsLair/duality/blob/f72221bc75e27a2450d96915adb3513c5df94284/Test/Core/Components/InitializableEventReceiver.cs), which has a similar role in testing scenarios.